### PR TITLE
New issnip indicators

### PR DESCRIPTION
**Pull request by emord**
@czue since you're reviewing htis code anyways

Can ignore the third commit when reviewing as it's just making whitespace consistent

Currently just adds the AWW name and their number in the locations table.

Will likely add the other two indicators to this PR (added migration, just needs the logic), but this is ready to go

### CODE REVIEW

File: custom/icds_reports/migrations/sql_templates/create_functions.sql [hunk 1]
```diff
@@ -27,8 +27,10 @@ CREATE OR REPLACE FUNCTION update_location_table() RETURNS VOID AS
 $BODY$
 DECLARE
 	_ucr_location_table text;
+  _ucr_aww_tablename text;
```
Comment (esoergel) [inline custom/icds_reports/migrations/sql_templates/create_functions.sql]: nit: looks like this is mixing tabs and spaces - alignment is off in the github UI

Comment (esoergel) [reply custom/icds_reports/migrations/sql_templates/create_functions.sql]: Haha, oh, just saw the next commit.

**Review by esoergel — APPROVED** [review]

File: custom/icds_reports/ucr/data_sources/dashboard/pregnant_tasks.json [hunk 2]
```diff
@@ -32,7 +32,7 @@
             "type": "property_name",
             "property_name": "tasks_type"
           },
-          "property_value": "pregnant"
+          "property_value": "pregnancy"
```
Comment (emord) [inline custom/icds_reports/ucr/data_sources/dashboard/pregnant_tasks.json]: was wondering why this data source wasn't populating. this was correct in the spec, but wrong in an email :/

**Review by emord — COMMENTED** [review]
Adding columns in a CSV file is extremely noisy. Need to figure out something better for these tests. The actual code changed is fairly small in comparison

**Review by esoergel — APPROVED** [review]

File: custom/icds_reports/migrations/sql_templates/create_functions.sql [hunk 3]
```diff
@@ -295,13 +295,42 @@ BEGIN
...
+      'ut.due_list_date_4g_vit_a_1 BETWEEN ' || quote_literal(_start_date) || ' AND ' || quote_literal(_end_date) || ' OR ' ||
+      'ut.due_list_date_5g_vit_a_2 BETWEEN ' || quote_literal(_start_date) || ' AND ' || quote_literal(_end_date) || ' OR ' ||
+      'ut.due_list_date_6g_vit_a_3 BETWEEN ' || quote_literal(_start_date) || ' AND ' || quote_literal(_end_date) || ' OR ' ||
+      'ut.due_list_date_6g_vit_a_4 BETWEEN ' || quote_literal(_start_date) || ' AND ' || quote_literal(_end_date) || ' OR ' ||
+      'ut.due_list_date_6g_vit_a_5 BETWEEN ' || quote_literal(_start_date) || ' AND ' || quote_literal(_end_date) || ' OR ' ||
+      'ut.due_list_date_6g_vit_a_6 BETWEEN ' || quote_literal(_start_date) || ' AND ' || quote_literal(_end_date) || ' OR ' ||
+      'ut.due_list_date_6g_vit_a_7 BETWEEN ' || quote_literal(_start_date) || ' AND ' || quote_literal(_end_date) || ' OR ' ||
+      'ut.due_list_date_6g_vit_a_8 BETWEEN ' || quote_literal(_start_date) || ' AND ' || quote_literal(_end_date) || ' OR ' ||
+      'ut.due_list_date_7g_vit_a_9 BETWEEN ' || quote_literal(_start_date) || ' AND ' || quote_literal(_end_date) ||
```
[the remark this answers is not in the document]
Comment (emord) [reply custom/icds_reports/migrations/sql_templates/create_functions.sql]: I haven't thought about a template.

The next step was going to be defining the tables [here](https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/models.py#L769) and queries [here](https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/utils/aggregation.py) like the other models I'm creating.

The dream is having aggregate tables be first class members of UCR, but that's also going to be dependent on cory's ML work I think and we'll need to spec it out more.

Happy to hear any other intermediate ideas you have though

File: custom/icds_reports/models.py [hunk 4]
```diff
@@ -121,6 +123,8 @@ class AggAwcMonthly(models.Model):
     district_map_location_name = models.TextField(blank=True, null=True)
     state_map_location_name = models.TextField(blank=True, null=True)
     month = models.DateField(blank=True, null=True)
+    aww_name = models.DateField(blank=True, null=True)
+    contact_phone_number = models.DateField(blank=True, null=True)
```
Comment (lwyszomi) [inline custom/icds_reports/models.py]: @czue @sravfeyn  Do you know why ```aww_name``` and ```contact_phone_numer``` are ```DateField```

Comment (czue) [reply custom/icds_reports/models.py]: I do not, nor can I fathom an explanation. Guessing it's a mistake?

Comment (lwyszomi) [reply custom/icds_reports/models.py]: @czue I also guessing it's a mistake. I will fix this when I will send PR with changes to the Monthly Register report.

Comment (czue) [reply custom/icds_reports/models.py]: that'd be great, thanks!

**Pull request merged**